js/src/components/contact-information/contact-information-preview-card.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, warning as warningIcon } from '@wordpress/icons';
+import { getPath, getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import AccountCard from '.~/components/account-card';
+import AppButton from '.~/components/app-button';
+import './contact-information-preview-card.scss';
+
+/**
+ * Renders a component with a given store address.
+ * In preview mode, meaning there will be no refresh button, just the edit link.
+ *
+ * @param {Object} props React props
+ * @param {import('.~/components/account-card').APPEARANCE}  props.appearance
+ * @param {string} props.editHref URL where Edit button should point to.
+ * @param {string} props.editEventName Tracing event name used when the "Edit" button is clicked.
+ * @param {boolean} props.loading Set ti `true` if the card should be rendered in the loading state.
+ * @param {JSX.Element} props.content Main content of the card to be rendered once the data is loaded.
+ * @param {string} [props.warning] Warning title, to be used instead of the default one.
+ * @return {JSX.Element} Filled AccountCard component.
+ */
+export default function ContactInformationPreviewCard( {
+	editHref,
+	editEventName,
+	loading,
+	content,
+	appearance,
+	warning,
+} ) {
+	const { subpath } = getQuery();
+	const editButton = (
+		<AppButton
+			isSecondary
+			iconSize={ 16 }
+			iconPosition="right"
+			href={ editHref }
```

Comment:
Thanks for catching this!

It seems to be linked to a more generic problem. Another one in the batch of WC vs WP Links.
I created [an issue](https://github.com/woocommerce/woocommerce-admin/issues/7725) in WC-admin repo.

To me `<Link><Button>` still has a serious A11Y issue, with being seen as "Edit - button" instead of "Edit - visited link".
I'll create an issue in our repo as well, to solve it somehow for all internal button links.

Reply:
`<Link>Button>` -> `<a><button>` is also an invalid HTML, so I'd leave it as is, and try to fix it sooner than later in #1027, by cloning wc-admins `wcAdminLinkHandler` to `AppButton`.

Reply:
> try to fix it sooner than later in #1027, by cloning wc-admins `wcAdminLinkHandler` to `AppButton`.

Thanks! I also think it's feasible.